app/Models/User.php

```diff
@@ -41,4 +43,14 @@ class User extends Authenticatable
     protected $casts = [
         'email_verified_at' => 'datetime',
     ];
+
+    /**
+     * Get the department the user belongs to
+     *
+     * @return BelongsTo
+     */
+    public function department()
+    {
+        return $this->belongsTo(Department::class);
```

Comment:
Make a habit of defining the foreign key as second param. The way you become clear about how relationship mapping is executed behind the scene.